_assets/build/testnet-flags.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/sh
-
-set -e
-
-if [ ! -f "_assets/build/testnet-flags.sh" ]; then
-    echo "$0 must be run from the root of the repository."
-    exit 2
-fi
-
-# set gitCommit when running from a Git checkout.
-if [ -f ".git/HEAD" ]; then
-    echo "-ldflags '-X github.com/status-im/status-go/geth/params.UseMainnetFlag=false -X main.buildStamp=`date -u '+%Y-%m-%d.%H:%M:%S'` -X main.gitCommit=$(git rev-parse HEAD)'";
```

Comment:
I would keep this file for now but remove `-X github.com/status-im/status-go/geth/params.UseMainnetFlag=false`.

Reply:
As mentioned above this part could be direct part of the Makefile.